besu/src/test/java/org/hyperledger/besu/RunnerTest.java

```diff
@@ -378,8 +380,6 @@ private void syncFromGenesis(final SyncMode mode, final GenesisConfigFile genesi
         runnerBehind.close();
         runnerBehind.awaitStop();
       }
-      runnerAhead.close();
```

Comment:
Ok this one was a huge pain to troubleshoot. Closing runnderBehind closes its vertx, which is shared with runnerAhead. Attempting to close an already closed vertx throws an exception.